docker-compose.yml

```diff
@@ -20,6 +22,30 @@ services:
       - 27017:27017
     networks:
       - ldes
+  pyroscope:
+    image: grafana/pyroscope:latest
+    ports:
+      - 4040:4040
+    networks:
+      - ldes
+
+  grafana:
+    image: grafana/grafana:main
+    ports:
+      - 3000:3000
+    environment:
+      - GF_FEATURE_TOGGLES_ENABLE=flameGraph
+    networks:
+      - ldes
+  jaeger:
+    image: jaegertracing/all-in-one:1.37
+    environment:
+      - COLLECTOR_OTLP_ENABLED=true
+    ports:
+      - "6831:6831/udp"
+      - "16686:16686"
+      - "4319:4317"
+      - "4320:4318"
+      - "14250:14250"
```

Comment:
Would leave this out.. Grafana is definitely not essential for LDES Server config. and why Jaeger for your story?